modules/gitops-configuring-a-cluster-scoped-instance-to-manage-rollouts-resources.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * argo_rollouts/using-cluster-scoped-rollouts-instance-to-manage-rollouts-resources.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-configuring-a-cluster-scoped-instance-to-manage-rollouts-resources_{context}"]
+= Configuring a cluster-scoped rollouts instance to manage rollouts resources 
+
+To configure a cluster-scoped rollouts instance for managing rollouts resources, add the `CLUSTER_SCOPED_ARGO_ROLLOUTS_NAMESPACES` environment variable in the `Subscription` resource. This variable contains a list of user-defined namespaces which can be configured for a cluster-scoped rollouts installation.
+
+[NOTE]
+====
+You can only create a cluster-scoped rollouts instance if the `NAMESPACE_SCOPED_ARGO_ROLLOUTS` variable is set to `false`.
```

Comment:
Can add a line like to the end, like: If `NAMESPACE_SCOPED_ARGO_ROLLOUTS` variable is not defined, it is assumed to be false.